### ISSUE TITLE
Zoom not working as it used to

### ISSUE THREAD

**Issue by maddy25** [highcharts/highcharts#13774]
#### Expected behaviour
XY zoom should work
#### Actual behaviour

Nothing happens, the bottom scroll works but zooming using mouse is not working.
#### Live demo with steps to reproduce
http://jsfiddle.net/6qewt5ay/

#### Product version
v8.1.2 highcharts

#### Affected browser(s)
Chrome (have not tested in anyother browser)

**Comment by pawelfus** [comment]
Hi @maddy25 

You need to enable zooming in your chart first, see [`chart.zoomType`](https://api.highcharts.com/highstock/chart.zoomType). Demo: http://jsfiddle.net/BlackLabel/azp8mbx9/

**Comment by maddy25** [comment]
Yes I understand that, in my actual codebase I have it (zoom) enabled but still not working. I lowered the highchart version to v7.2.1 when I saw 8.1.2 was not working.

**Comment by pawelfus** [comment]
Boost shouldn't affect zooming. You might have found a bug @maddy25 but without live demo we won't be able to debug this (neither fix it).

**Comment by maddy25** [comment]
https://codesandbox.io/s/angular-qo4zf?file=/src/app/chart.component.ts
Plz see this, it will be great if you could point out why zoom is not working in this case. @pawelfus

**Comment by pawelfus** [comment]
Thank you for the demo! I can see this clearly now.

Simplified demo:
- https://jsfiddle.net/BlackLabel/dc3yogt6/

I'm not sure if this is a bug. Let me explain: it's caused by `tooltip.style.pointerEvents = "auto"`. When you set this, tooltip will gather all events. And when `split` option is enabled, then tooltip's container will cover most part of the chart, so you won't be able to zoom in, show tooltip again etc.

When removed, works fine: https://codesandbox.io/s/angular-4id40
Or:
- use non-split tooltip: https://jsfiddle.net/BlackLabel/ogsv6m5u/1/
- disable `tooltip.outside` option: https://jsfiddle.net/BlackLabel/ogsv6m5u/2/ (works exactly the same as v7.2.0)

It used to "work fine" in v7.2.0, because `tooltip.split` did not support `tooltip.outside`; `tooltip.outside` was always disabled for split tooltip.


*Internal note:*
v7.2.0: https://jsfiddle.net/BlackLabel/ogsv6m5u/

Bisected to commit f13c497470 - which basically means implementation `split` + `outside`.

For now, I'm marking this as "undecided", for further internal discussion.

**Comment by maddy25** [comment]
Ok thank you for the explanation, I will set outside to false.
